src/comb/template.clj

```diff
@@ -18,23 +19,33 @@
         ")?"
         "(.*)\\z")))
 
-(defn emit-string [s]
-  (print "(print " (pr-str s) ")"))
+(defn emit-string
+  ([s]
+     (print "(print " (pr-str s) ")"))
+  ([s emit-next-line-break?]
+     (if emit-next-line-break?
```

Comment:
I think there's a better way of doing this. We can change the regular expression instead:

```clojure
(def delimiters ["<%" "%>"])
(def trimmer    "-")
 
(def parser-regex
  (let [[s e] delimiters, t trimmer]
    (re-pattern
     (str "(?s)\\A"
          "(?:" "(.*?)"
          "(?:" "\\s*" s t "|" s ")"
          "(.*?)"
          "(?:" t e "\\s*" "|" e ")"
          ")?"
          "(.*)\\z"))))
```

So we match `\s*<%-` or `<%` at the beginning, and `-%>\s*` or `%>` at the end. This also allows for trimming whitespace at the beginning.